samples/java/cucumber-selenium/src/test/java/demo/StepDefinitions.java

```diff
@@ -0,0 +1,56 @@
+package demo;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class StepDefinitions {
+	private WebDriver driver;
+
+	@Given("^I navigate to the testrail website$")
+	public void i_navigate_to_the_testrail_website() {
+		System.out.println("Given statement executed successfully");
+
+		WebDriverManager.chromedriver().setup();
```

Comment:
all this webdriver initialization I think it would make sense to be inside a Before hook.
https://cucumber.io/docs/cucumber/api/?lang=java#hooks

Reply:
updated/added before hook.